MSAL/src/native_auth/controllers/credentials/MSALNativeAuthCredentialsController.swift

```diff
@@ -194,7 +194,9 @@ final class MSALNativeAuthCredentialsController: MSALNativeAuthTokenController,
                 return .init(.failure(error), correlationId: context.correlationId())
             }
             userAccountResult.refreshData(authTokens: authTokens, account: account)
-            return .init(.success(MSALNativeAuthTokenResult(authTokens: authTokens)),
+            return .init(.success(MSALNativeAuthTokenResult(accessToken: authTokens.accessToken.accessToken,
+                                                            scopes: authTokens.accessToken.scopes?.array as? [String] ?? [],
```

Comment:
MSAL ObjC allocates an ordered set, initialised with the passed scopes. So it will never be null: " scopes:[[NSOrderedSet alloc] initWithArray:parameters.scopes copyItems:YES]"